themes/default/content/registry/packages/eks/_index.md

```diff
@@ -4,7 +4,7 @@ meta_desc: Learn how you can use Pulumi's Amazon EKS component to set up an AWS
 layout: overview
 ---
 
-Amazon EKS is a Pulumi Component that creates and manages the resources necessary to run an [EKS Kubernetes cluster](https://aws.amazon.com/eks/) in AWS. Use this component to quickly set up an EKS cluster in just a few lines of code.
+Amazon EKS is a Pulumi Component that creates and manages the resources necessary to run an [EKS Kubernetes cluster](https://aws.amazon.com/eks/) in AWS. Use this component to quickly set up an EKS cluster in just a few lines of code. This component exposes the [Crosswalk for AWS](({{<relref "/docs/guides/crosswalk/aws">}})) functionality documented in the [Pulumi Amazon EKS guide]({{< relref "/docs/guides/crosswalk/aws/eks/" >}}).
```

Comment:
@susanev the relref to the crosswalk docs page is not working here for some reason. do you know why?

Reply:
i dont think we are supposed to be using relref anymore i would instead use the regular link. i don't think the actual link will work for you until this gets to docs though, unless you pull this package down locally.

`[Crosswalk for AWS](/docs/guides/crosswalk/aws/)`